sentry_sdk/integrations/opentelemetry/span_processor.py

```diff
@@ -98,6 +98,9 @@ def on_start(self, otel_span, parent_context=None):
         if not hub:
             return
 
+        if not hub.client or (hub.client and not hub.client.dsn):
```

Comment:
I assume this also takes care of invalid DSN or invalid DSN is already handled somewhere else and doesn't make it here?

Reply:
Good point! I have added a new check for invalid dsn (see commits below)